docs/source/reference/concept.md

```diff
@@ -0,0 +1,43 @@
+# Concept and FAQs
+
+There are a few terms used in `mosec`.
+
+- `worker`: a Python process that executes the `forward` method (inherit from [`mosec.Worker`](mosec.worker.Worker))
+- `stage`: one processing unit in the pipeline, each stage contains several `worker` replicas
```

Comment:
The `pipeline` concept is not mentioned yet, also, a single-stage server does not need a pipeline. Maybe mention the pipeline concept in its own bullet point and illustrate more? 

OR

mention the pipeline concept earlier (since ingress and egress also refer to it) and state a single-stage server is just a special case.

Reply:
But the `pipeline` concept also contains the `stage`. Circular Import :rofl:.